docs/advanced/navigating-politeia-data.md

```diff
@@ -0,0 +1,75 @@
+
+
+
+
+# <img class="dcr-icon" src="/img/dcr-icons/Politeia.svg" /> Navigating Politeia Data
+
+Data from the [proposals.decred.org](https://proposals.decred.org/) site is stored in [this repository](https://github.com/decred-proposals/mainnet). This page gives an overview of how to navigate that repository and how to interpret the contents of its files.
+
+For the purpose of this guide I will be using [this proposal](https://github.com/decred-proposals/mainnet/tree/master/c68bb790ba0843980bb9695de4628995e75e0d1f36c992951db49eca7b3b4bcd/) as an example. Links from [proposals.decred.org](https://proposals.decred.org/) will take you one layer deeper to the [most recent version of the proposal](https://github.com/decred-proposals/mainnet/tree/master/c68bb790ba0843980bb9695de4628995e75e0d1f36c992951db49eca7b3b4bcd/2). 
+
+![The root of the mainnet repository. * = proposal folders, A = anchoring data](/img/politeia/mainnet-pi-repo.PNG)
+
+The root of the mainnet repository. * = proposal folders, A = anchoring data.
+
+The folder and file marked A relate to the anchoring of Politeia data to the Decred blockchain through dcrtime. This guide will not describe those files in detail as they are not likely to be of interest for analysis. They are functional and facilitate Politeia's transparent censorship.
+
+Within each proposal folder there are sequentially numbered sub-folders for each version of the proposal.
+
+![](/img/politeia/example-proposal.PNG)
+
+Within the version number folder all relevant data for that proposal can be found.
+
+![Folder for version 2 of the example proposal](/img/politeia/prop-version2.png)
+
+* **payload** folder contains an index.md file which has the full text of the proposal, and any images associated with the proposal
+* **plugins/decred** folder contains comments.journal (comments and up/down votes data) and, for proposals that have been opened for voting, a balltot.journal file which contains ticket vote data
+* **00.metadata.txt**: data about proposal submission. timestamp, pubkey of submitter, title, signature
+* **02.metadata.txt**: data about admin approval for display. timestamp, pubkey of admin
+* **13.metadata.txt:** data about proposal owner authorizing the start of voting. timestamp, owner pubkey, transaction hash. Where proposal owner has authorized and then rescinded authorization this will appear in multiple commits.
```

Comment:
The content lists in these bullet points (ex. timestamp, owner pubkey, transaction hash) make these hard to read.  I would suggest either including these as sub-bullets.

```suggestion
* **13.metadata.txt:** data about proposal owner authorizing the start of voting. Where proposal owner has authorized and then rescinded authorization this will appear in multiple commits.
  * timestamp
  * pubkey of author
  * transaction hash 
```

Reply:
Good suggestion.